package.json

```diff
@@ -43,30 +47,11 @@
     "husky": "^0.14.3",
     "node-sass": "^4.5.3",
     "react-dev-utils": "^4.0.0",
+    "react-intl-translations-manager": "^5.0.1",
     "sass-loader": "^6.0.6",
     "style-loader": "^0.18.2",
     "webpack": "^3.5.5",
     "webpack-dev-server": "^2.7.1",
     "webpack-merge": "^4.1.0"
-  },
-  "babel": {
-    "presets": [
-      [
-        "env",
-        {
-          "targets": {
-            "browsers": [
-              "last 2 versions",
-              "ie 11"
-            ]
-          }
-        }
-      ],
-      "babel-preset-react"
-    ],
-    "plugins": [
-      "transform-object-rest-spread",
-      "transform-class-properties"
-    ]
```

Comment:
I think @MichaelRoytman is yanking out `.babelrc` in his work; I replayed here b/c all the tutorials assumed that setup.